packages/shared/src/components/filters/TagButton.tsx

```diff
@@ -78,7 +78,7 @@ const FollowTagButton = ({
   <GenericTagButton
     {...props}
     className={classNames('btn-tag', className)}
-    icon={<PlusIcon className="ml-2 text-xl transition-transform rotate-0" />}
```

Comment:
I think we used the transform here to do the undo transform could you verify that still animates? (so both ways)

Reply:
Yeah, you are right, it is so tiny that I haven't initially noticed it